src/Uno.UWP/Devices/Sensors/Accelerometer.wasm.cs

```diff
@@ -82,7 +84,6 @@ private void DetachDeviceMotion()
 		/// <param name="y">Accelerometer Y</param>
 		/// <param name="z">Accelerometer Z</param>
 		/// <returns>0 - needed to bind method from WASM</returns>
-		[Preserve]
```

Comment:
^ do you still care for the legacy Xamarin SDK ?
because the newer attribute(s) is/are not supported

Reply:
Actually https://github.com/unoplatform/uno/blob/master/src/Uno.Foundation/PreserveAttribute.cs is being used 🤔

Reply:
That's fine. The old linker accepts any `[Preserve]` attribute (by name, not by type) so it can work with .NET Standard (or older FX Xamarin had to support)

Reply:
Also I think the correct syntax should have been `[Preserve(Conditional = true)]`

This means the decorated member will be preserved if the type is used.
If the type is not marked then the member will not be preserved.

Reply:
The preserve attribute is interpreted by a custom step in the xamarin tooling only, so we can remove that one altogether for wasm targets.

Reply:
What this does, nobody knows, I'll delete it :)

Reply:
oh, missed that it was wasm only :)
yeah, delete it!